openmc/deplete/stepresult.py

```diff
@@ -198,6 +199,34 @@ def distribute(self, local_materials, ranges):
         new.rates = [r[ranges] for r in self.rates]
         return new
 
+    def get_material(self, mat_id):
+        """Return material object for given depleted composition
+
+        .. versionadded:: 0.14.0
+
+        Parameters
+        ----------
+        mat_id : str
+            Material ID as a string
+
+        Returns
+        -------
+        openmc.Material
+            Equivalent material
+        """
+        with warnings.catch_warnings():
+            warnings.simplefilter('ignore', openmc.IDWarning)
+            material = openmc.Material(material_id=int(mat_id))
+        vol = self.volume[mat_id]
+        for nuc, _ in sorted(self.index_nuc.items(), key=lambda x: x[1]):
+            atoms = self[0, mat_id, nuc]
+            if atoms < 0.0:
+                continue
+            atom_per_bcm = atoms / vol * 1e-24
+            material.add_nuclide(nuc, atom_per_bcm)
+        material.volume = vol
```

Comment:
It appears that this material object returned has no density set.

Reply:
Could ```set_density('sum')``` perhaps help set the density here

Reply:
That is actually the default when you don't specify anything :smile: would you prefer I put it in explicitly?

Reply:
perhaps we can just remove ```set_density('sum')``` from this other line then
https://github.com/openmc-dev/openmc/blob/c6a0381de74ee4c2b554a16a90bd2019322dde70/openmc/deplete/results.py#L446

Reply:
That line has a purpose -- the material that is being modified might already have a density set differently. It was actually added as a bug fix in #1946

Reply:
no changes needed then